sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/GenericArrayData.scala

```diff
@@ -35,6 +35,7 @@ private object GenericArrayData {
 
 class GenericArrayData(val array: Array[Any]) extends ArrayData {
 
+  def this(size: Int) = this(new Array[Any](size))
```

Comment:
What is it for BTW?

Reply:
sorry, it's from previous solution, I removed it